src/Lean/Server/Watchdog.lean

```diff
@@ -368,8 +368,9 @@ def handleReference (p : ReferenceParams) : ServerM (Array Location) := do
 def handleWorkspaceSymbol (p : WorkspaceSymbolParams) : ServerM (Array SymbolInformation) := do
   let references ← (← read).references.get
   let srcSearchPath := (← read).srcSearchPath
-  let symbols ← references.definitionsMatching srcSearchPath (maxAmount := some 100)
-    fun name => containsCaseInsensitive p.query $ name.toString
+  let symbols ← references.definitionsMatching srcSearchPath (maxAmount := some 100) fun
+    | Name.str _ s _ => containsCaseInsensitive p.query s
+    | _              => false
```

Comment:
Hmm, I always liked to do `ctrl-p # nat.ltlt` to find nat-specific lemmas.

Reply:
Yeah, I removed it again, it's not too bad that way. It should get better when we prioritize consecutive matches.